plugins/maciVoting/components/PollCard.tsx

```diff
@@ -92,13 +90,7 @@ const PollCard = ({ pollId }: { pollId: bigint }) => {
             </div>
           )}
           <p>The voting period has ended. Here are the results:</p>
-          <VoteResultCard
-            results={{
-              yes: Number(results[0].value),
-              no: Number(results[1].value),
-              abstain: Number(results[2].value),
-            }}
-          />
+          <VoteResultCard pollId={pollId} />
```

Comment:
Send the poll id to get the results inside the `<VoteResultCard>`. I initially thought this fixed the problem of showing the right results right away after finalizing the poll but it did not fix it.

I think it is cleaner to read the results inside this component (we could even add more logic to refetch them every once in a while